scala_pekko_bench/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM hseeberger/scala-sbt:11.0.7_1.3.13_2.11.12 as BUILDER
```

Comment:
I think https://hub.docker.com/r/sbtscala/scala-sbt/tags is better choice.

Reply:
Thanks, what about after this got merged and you submitted a new pr for it？